From f0809ea6dc80f89e55eb62f46f3e43b2f5b08765 Mon Sep 17 00:00:00 2001
From: Ruslan Bay <67730802+ruslanbay@users.noreply.github.com>
Date: Sat, 10 Oct 2026 13:06:50 +0200
Subject: [PATCH 12/13] media: ipu6: Enable IPU4P PCI device

Register IPU4P in the PCI ID table after its required MMU, buttress,
firmware, ISYS, and CSI PHY support is present.

Signed-off-by: Ruslan Bay <ruslanbey@proton.me>
---
 include/media/ipu6-pci-table.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/media/ipu6-pci-table.h b/include/media/ipu6-pci-table.h
index 7369d46621af..09b2ea58bc1f 100644
--- a/include/media/ipu6-pci-table.h
+++ b/include/media/ipu6-pci-table.h
@@ -27,6 +27,7 @@ static const struct pci_device_id ipu6_pci_tbl[] = {
 	{ PCI_VDEVICE(INTEL, PCI_DEVICE_ID_INTEL_IPU6EP_MTL) },
 	{ PCI_VDEVICE(INTEL, PCI_DEVICE_ID_INTEL_IPU7) },
 	{ PCI_VDEVICE(INTEL, PCI_DEVICE_ID_INTEL_IPU7P5) },
+	{ PCI_VDEVICE(INTEL, PCI_DEVICE_ID_INTEL_IPU4P) },
 	{ }
 };
 
-- 
2.55.0

